examples/camel-example-ceylon/src/main/ceylon/org/apache/camel/example/module.ceylon

```diff
@@ -0,0 +1,22 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+native("jvm")
+module org.apache.camel.example "1.0.0" {
+
+    import maven:"org.apache.camel:camel-core" "2.22.0-SNAPSHOT";
```

Comment:
If you don't want to hardcode camel components version inside .ceylone module, it's possible to use some template file and fill it on the build stage inside maven.

Reply:
I tried a few things but it didn't work. The closest one I got was the Filtering capability of the Maven Resources plugin.(https://maven.apache.org/plugins/maven-resources-plugin/examples/filter.html). It works well in what we want it to do: pointing at the package where the module.ceylon is included it is indeed changing the ${project.version} and store it under target/classes. Problem is that the Ceylon compiler (under the plugin) is not considering the target entry but rather the one under sources. I tried with the resources config of the Ceylon plugin too to no avail